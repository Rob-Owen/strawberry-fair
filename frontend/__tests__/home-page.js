import React from "react"
import renderer from "react-test-renderer"
import HomePage from "../src/templates/home-page"


describe("Home Page", () =>
  it("renders correctly", () => {
    const tree = renderer.create(<HomePage />).toJSON()
    expect(tree).toMatchSnapshot()
  }))