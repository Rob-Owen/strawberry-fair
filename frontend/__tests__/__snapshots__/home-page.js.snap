// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correctly 1`] = `
<div
  className="container"
>
  <header>
    <nav
      className="navbar is-fixed-top"
    >
      <div
        className="navbar-brand"
      >
        <a
          className="navbar-item"
          href="/"
        >
          <img
            alt="Strawberry Fair Logo"
            src="/img/1-line-logo.png"
            style={
              Object {
                "borderRadius": "5px",
              }
            }
          />
        </a>
        <a
          className="navbar-burger burger "
          data-target="navigationBar"
          onClick={[Function]}
        >
          <span />
          <span />
          <span />
        </a>
      </div>
      <ul
        className="navbar-menu "
        id="navigationBar"
      >
        <div
          className="navbar-start"
        >
          <li
            className="navbar-item has-dropdown "
          >
            <div
              className="dropdown"
            >
              <div
                className="dropdown-trigger"
              >
                <div>
                  <button
                    className="button"
                    onClick={[Function]}
                  >
                    About Us
                  </button>
                </div>
              </div>
              <ul
                className="navbar-dropdown is-hidden-touch"
              >
                <li
                  className="navbar-item"
                >
                  <a
                    className="dropdown-item"
                    href="/"
                  >
                    Frequently Asked Questions
                  </a>
                </li>
                <li
                  className="navbar-item"
                >
                  <a
                    className="dropdown-item"
                    href="/"
                  >
                    About the Fair
                  </a>
                </li>
                <li
                  className="navbar-item"
                >
                  <a
                    className="dropdown-item"
                    href="/"
                  >
                    Organisation
                  </a>
                </li>
                <li
                  className="navbar-item"
                >
                  <a
                    className="dropdown-item"
                    href="/"
                  >
                    History
                  </a>
                </li>
                <li
                  className="navbar-item"
                >
                  <a
                    className="dropdown-item"
                    href="/"
                  >
                    Environment
                  </a>
                </li>
                <li
                  className="navbar-item"
                >
                  <a
                    className="dropdown-item"
                    href="/"
                  >
                    Bucket Collection Partners
                  </a>
                </li>
                <li
                  className="navbar-item"
                >
                  <a
                    className="dropdown-item"
                    href="/"
                  >
                    Get Involved
                  </a>
                </li>
              </ul>
            </div>
          </li>
          <li
            className="navbar-item has-dropdown "
          >
            <div
              className="dropdown"
            >
              <div
                className="dropdown-trigger"
              >
                <div>
                  <button
                    className="button"
                    onClick={[Function]}
                  >
                    Areas & Events
                  </button>
                </div>
              </div>
              <ul
                className="navbar-dropdown is-hidden-touch"
              >
                <li
                  className="navbar-item"
                >
                  <a
                    className="dropdown-item"
                    href="/calendar"
                  >
                    Calendar
                  </a>
                </li>
              </ul>
            </div>
          </li>
          <li
            className="navbar-item has-dropdown "
          >
            <div
              className="dropdown"
            >
              <div
                className="dropdown-trigger"
              >
                <div>
                  <button
                    className="button"
                    onClick={[Function]}
                  >
                    News
                  </button>
                </div>
              </div>
              <ul
                className="navbar-dropdown is-hidden-touch"
              />
            </div>
          </li>
          <li
            className="navbar-item has-dropdown "
          >
            <div
              className="dropdown"
            >
              <div
                className="dropdown-trigger"
              >
                <div>
                  <button
                    className="button"
                    onClick={[Function]}
                  >
                    Traders
                  </button>
                </div>
              </div>
              <ul
                className="navbar-dropdown is-hidden-touch"
              />
            </div>
          </li>
          <li
            className="navbar-item has-dropdown "
          >
            <div
              className="dropdown"
            >
              <div
                className="dropdown-trigger"
              >
                <div>
                  <button
                    className="button"
                    onClick={[Function]}
                  >
                    Support the Fair
                  </button>
                </div>
              </div>
              <ul
                className="navbar-dropdown is-hidden-touch"
              />
            </div>
          </li>
          <li
            className="navbar-item has-dropdown "
          >
            <div
              className="dropdown"
            >
              <div
                className="dropdown-trigger"
              >
                <div>
                  <button
                    className="button"
                    onClick={[Function]}
                  >
                    Contact
                  </button>
                </div>
              </div>
              <ul
                className="navbar-dropdown is-hidden-touch"
              />
            </div>
          </li>
        </div>
      </ul>
    </nav>
  </header>
  <main>
    <section>
      <h1
        className="title"
      />
    </section>
  </main>
  <footer
    className="footer"
  >
    <div
      className="tile is-ancestor"
    >
      <div
        className="tile is-parent"
      >
        <div
          className="tile is-child box"
        >
          <h3
            className="title"
          >
            Placeholder
          </h3>
        </div>
      </div>
      <div
        className="tile is-parent"
      >
        <div
          className="tile is-child box"
        >
          <h3
            className="tile is-content title"
          >
            Follow
          </h3>
          <div
            className="tile is-parent"
          >
            <div
              className="tile is-child"
            >
              <a
                className="image is-48x48"
                href="https://www.facebook.com/strawberryfair"
              >
                <img
                  alt="Facebook"
                  src="img/facebook-logo.png"
                  style={
                    Object {
                      "borderRadius": "5px",
                    }
                  }
                />
              </a>
            </div>
            <div
              className="tile is-child"
            >
              <a
                className="image is-48x48"
                href="https://twitter.com/strawberry_fair"
              >
                <img
                  alt="Twitter"
                  src="img/twitter-logo.png"
                  style={
                    Object {
                      "borderRadius": "5px",
                    }
                  }
                />
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="tile is-parent"
      >
        <div
          className="tile is-child box"
        >
          <h3
            className="title"
          >
            Contact
          </h3>
          PO Box 1261 
          <br />
           Cambridge 
          <br />
           CB1 0YJ 
          <br />
          <br />
          <a
            href="mailto:enquiries@strawberry-fair.org.uk"
          >
            enquiries@strawberry-fair.org.uk
          </a>
        </div>
      </div>
    </div>
    <div
      className="tile is-ancestor"
    >
      <div
        className="tile is-parent"
      >
        <div
          className="tile is-child box content"
        >
          Â© Strawberry Fair 2017 | 
          <a
            href="/"
          >
             Cookie policy
          </a>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
